package/MDAnalysis/converters/RDKitParser.py

```diff
@@ -72,6 +73,18 @@
 logger = logging.getLogger("MDAnalysis.converters.RDKitParser")
 
 
+def _rdkit_chiral_to_RS(val):
+    # convert RDKit chirality tag to R/S/''
+    from rdkit import Chem
+
+    if val == Chem.ChiralType.CHI_TETRAHEDRAL_CW:
+        return 'R'
+    elif val == Chem.ChiralType.CHI_TETRAHEDRAL_CCW:
+        return 'S'
+    else:
+        return ''
```

Comment:
So it turns out that `Chem.ChiralType.CHI_TETRAHEDRAL_CW` and CCW are based on the atoms order, which are not guaranteed to match with the CIP order according to [this thread](https://www.mail-archive.com/rdkit-discuss@lists.sourceforge.net/msg09255.html).

However, the chiral tags are available as an atom property when reading a molecule made from an isomeric SMILES or if `Chem.AssignStereochemistryFrom3D` was called, so everything is not lost!

```suggestion
def _rdkit_atom_to_RS(atom):
    """Fetches RDKit chiral tags"""
    try:
        return atom.GetProp("_CIPCode")
    except KeyError:
        return ""
```